db/migrate/20190912121620_create_conversations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateConversations < ActiveRecord::Migration[5.2]
+  def change
+    create_table(:conversations, &:timestamps)
+
+    create_table :conversation_users, bulk: true, id: false do |t|
```

Comment:
I know you removed this `id: false` in futures PR

Reply:
yes 😭  silly past me